test/system/500-networking.bats

```diff
@@ -12,7 +12,7 @@ load helpers
     random_2=$(random_string 30)
 
     HOST_PORT=8080
-    SERVER=http://localhost:$HOST_PORT
+    SERVER=http://127.0.0.1:$HOST_PORT
```

Comment:
This is completely weird. When running podman as root, 'localhost' no longer seems to work, at least with alpine: `wget` barfs with ECONNREFUSED. It works fine with rootless. I gave up trying to track it down, and explicit IPv4 home seems like a better idea anyway.

Reply:
Weird, but localhost is hard coded within container/image and podman to represent locally built images.